app/views/tweets/tweets.erb

```diff
@@ -0,0 +1,14 @@
+<% if logged_in?  %>
+       <h1>Welcome, <%= current_user.username %></h1>
+
+<%Tweet.all.each do |tweet| %>
```

Comment:
With an MVC design pattern, it is best practice to separate the concerns of Model calls in the controller. Example:

```ruby 
get '/tweets' do 
  @tweets = Tweet.all
end
``` 

and now you will have access to `<% @tweets.each do |tweet| %>`